codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/awssdk/customize/AwsSdkShimFileWriter.java

```diff
@@ -151,7 +151,16 @@ private void generateAwsSdkErrorToDafnyErrorBlock(
       // and this function should only cast to Opaque errors
       writer.write(
         """
-        return $L.Error_Opaque(obj=e, alt__text=repr(e))
+        return $L.Error_Opaque(obj=e, alt__text=_dafny.Seq(
+            repr(e).join(
+                [
+                    chr(int.from_bytes(pair, "big"))
+                    for pair in zip(
+                        *[iter(native_input.version.encode("utf-16-be"))] * 2
+                    )
+                ]
+            )
+        ))
```

Comment:
nit: You how, when you make a mistake, you have to fix every copy of it?

You may want to save future devs some time/effort by putting this copy and pasted block into a constant and referencing it.

Reply:
Done